django/contrib/admin/templatetags/admin_list.py

```diff
@@ -35,36 +36,37 @@
 
 
 @register.simple_tag
-def paginator_number(cl, i):
+def paginator_number(p, i):
```

Comment:
I'm aware you are replacing the `cl` abbreviation that is 2 letters with a similar-type abbreviation for pagination but `p` is probably too short. Could this be `pagination` instead of just `p`?